### PR TITLE
Reformat for better HTML

### PR DESCRIPTION
**Pull request by FiloSottile**
The `""` in the text format are not great but not the worst either, and we should embrace the future.

### CODE REVIEW

**Comment by FiloSottile** [comment]
It's pretty https://htmlpreview.github.io/?https://github.com/FiloSottile/draft-irtf-cfrg-ristretto255/blob/30e6be2e99d25fa69489d2929d30b7ac915cc5a5/draft-irtf-cfrg-ristretto255.html

File: draft-irtf-cfrg-ristretto255.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ The MAP function is defined on a 32-bytes string as:
    integer r in little-endian representation. Reduce r modulo p to
    obtain a field element t.
    * Note: similarly to [@RFC7748] field element decoding, the most
-     significant bit of the representations of r0 and r1 is masked.
+     significant bit of the representation of r is masked.
```
Comment (hdevalence) [inline draft-irtf-cfrg-ristretto255.md:444]: 👍

**Review by hdevalence — APPROVED** [review]
LGTM

**Comment by hdevalence** [comment]
I guess it's the tool but I'm surprised that the ``` are turned into `"` instead of just being passed through. Seems like a weird choice.

**Review by str4d — APPROVED** [review]

**Comment by FiloSottile** [comment]
> I guess it's the tool but I'm surprised that the ``` are turned into `"` instead of just being passed through. Seems like a weird choice.

Same. I could manually remove them, and upload a txt along with the xml, but seems like a lot of churn at least early on.

**Pull request merged**